demos/multi_channel_common/cpp/graph.cpp

```diff
@@ -189,32 +89,57 @@ void IEGraph::start(GetterFunc getterFunc, PostprocessingFunc postprocessingFunc
 }
 
 IEGraph::IEGraph(const InitParams& p):
-    perfTimerPreprocess(p.collectStats ? PerfTimer::DefaultIterationsCount : 0),
-    perfTimerInfer(p.collectStats ? PerfTimer::DefaultIterationsCount : 0),
-    confidenceThreshold(0.5f), batchSize(p.batchSize),
-    modelPath(p.modelPath),
-    cpuExtensionPath(p.cpuExtPath), cldnnConfigPath(p.cldnnConfigPath),
-    maxRequests(p.maxRequests) {
-    assert(p.maxRequests > 0);
+        perfTimerPreprocess(p.collectStats ? PerfTimer::DefaultIterationsCount : 0),
+        perfTimerInfer(p.collectStats ? PerfTimer::DefaultIterationsCount : 0),
+        confidenceThreshold(0.5f), batchSize(p.batchSize),
+        modelPath(p.modelPath),
+        postRead(p.postReadFunc) {
+    std::shared_ptr<ov::Function> model = core.read_model(modelPath);
+    if (model->get_parameters().size() != 1) {
+        throw std::logic_error("Face Detection model must have only one input");
+    }
+    const ov::Layout inLyout{"NHWC"};
+    model = ov::preprocess::PrePostProcessor(model).input(ov::preprocess::InputInfo()
+        .tensor(ov::preprocess::InputTensorInfo()
+            .set_element_type(ov::element::u8)
+            .set_layout(inLyout))
+        .preprocess(ov::preprocess::PreProcessSteps()
+            .convert_element_type(ov::element::f32)
+            .convert_layout("NCHW"))
+        .network(ov::preprocess::InputNetworkInfo().set_layout("NCHW"))
+    ).build();
+    ov::Shape inShape = model->input().get_shape();
+    // Set batch size
+    inShape[ov::layout::batch_idx(inLyout)] = batchSize;
+    model->reshape({{model->input().get_any_name(), inShape}});
 
-    postLoad = p.postLoadFunc;
-    initNetwork(p.deviceName);
+    if (postRead != nullptr)
+        postRead(model);
+    core.set_config({{"CPU_BIND_THREAD", "NO"}}, "CPU");
+    ov::runtime::ExecutableNetwork net = core.compile_model(model, p.deviceName, {{"PERFORMANCE_HINT", "THROUGHPUT"}});
```

Comment:
pls add optional PERFORMANCE_HINT_NUM_REQUESTS to give the OV hint on how requests will be inferred simultaneously

Reply:
Added